azure/functions_worker/logging.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import logging.handlers
+import sys
+
+
+logger = logging.getLogger('azure.functions_worker')
+error_logger = logger
```

Comment:
@elprans I'm seeing that the newly added `error_logger` is also logging to `stdout`. On further investigation it looks like that `error_logger` has the same configuration as the `logger`: 

![error_logger](https://user-images.githubusercontent.com/42051041/48574793-f0716f00-e8c4-11e8-8c3a-4e1f9b12207d.png)


I believe this is related to the assignment `error_logger = logger` in `logging.py`. When I modify this line to `error_logger = logging.getLogger('azure.functions_worker_errors')` (removing this from within the setup function and keeping everything else the same), I am seeing that the `error_logger` is logging correctly to `stderr`:

![error_logger2](https://user-images.githubusercontent.com/42051041/48575057-c8ced680-e8c5-11e8-820b-89021dc0f6af.png)

The `global` modifier is allowing for the correct modification of the `error_logger` but not allowing for assignment it seems. Your thoughts on what the issue might be and how to get around it?

Reply:
The `global` modifier is there exactly for the variable reassignment.  The actual issue is that `error_logger` is bound in global imports before `setup()` is ran.

Reply:
Could you elaborate a little more on what you mean by "global imports" here? Are you referring to the behavior when we import `error_logger` into `main.py` and `dispatcher.py` where it is getting bound to the same value as `logger`?

Reply:
Yes.  I'll amend the PR to fix this.